### PR TITLE
Kuz 631 internalengine for security repositories

### PR DESCRIPTION
**Pull request by ballinette**
Force the repositories to use the internalEngine instead of readEngine/writeWorker to persist users/roles/profiles into the database.

Also add parameters "from" and "size" to internalEngine.search() method

### CODE REVIEW

**Comment by benoitvidis** [comment]
:+1:

**Comment by stafyniaksacha** [comment]
Yeay +1 !

File: lib/api/core/models/repositories/README.md [hunk 1]
```diff
@@ -77,13 +71,13 @@ If no matching document could be found, an empty array is returned.
 Repository.prototype.search = function (filter, from, size, hydrate) {...}
```
Comment (j33f) [inline lib/api/core/models/repositories/README.md]: oops, still an "hydrate" parameter reference here

File: lib/api/core/models/repositories/README.md [hunk 2]
```diff
@@ -95,11 +89,11 @@ Number of hits to return (default: 20).
 
 * *Boolean* hydrate
 
-If hydrate is true, the function resolve a ResponseObject with a list of business objects. If hydrate is false, resolve a ResponseObject with documents directly from readEngine. (default: false).
+If hydrate is true, the function resolve a ResponseObject with a list of business objects. If hydrate is false, resolve a ResponseObject with documents directly from databaseEngine. (default: false).
```
Comment (j33f) [inline lib/api/core/models/repositories/README.md]: hydrate again

**Comment by j33f** [comment]
👍

**Pull request merged**